### PR TITLE
support xorg 1.20

### PR DESCRIPTION
**Pull request by nameless3**
path changed from `/usr/lib/xorg-server/Xorg` to `/usr/lib/Xorg`

### CODE REVIEW

**Comment by Earnestly** [comment]
It appears that `/usr/bin/Xorg` is actually the same binary as `/usr/lib/Xorg`.  Not only that but `/usr/bin/X` points to `/usr/bin/Xorg` now too.

There are no changelogs from upstream that I can see which explains this change yet so I'll have to read the commits.

This is pretty aggravating but Xorg isn't unknown for breaking things in backwards-incompatible ways before.

File: README [hunk 1]
```diff
@@ -22,7 +22,7 @@ INTRODUCTION
     * Corresponding xauth entries are unconditionally removed when the server
       exits.
     * The server uses the -noreset flag.
-    * /usr/lib/xorg-server/Xorg is used directly to bypass Xorg.wrap.
+    * /usr/lib//Xorg is used directly to bypass Xorg.wrap.
```
Comment (Earnestly) [inline README]: Can you clean up the doubled up `//` here?

**Comment by Earnestly** [comment]
Okay, this isn't going to be a hill I choose to die on.  I'd suggest switching the `exec /usr/lib/Xorg` to just use `exec Xorg` so we fall back on `PATH` resolution.

Then remove the note about `Xorg.wrap`.

This should be the most portable approach and if older releases of Xorg have to deal with the `wrap` nonsense, they can modify `/etc/X11/Xwrapper.conf` instead.

File: sx [hunk 2]
```diff
@@ -40,5 +40,5 @@ xauth add :"$tty" MIT-MAGIC-COOKIE-1 "$(od -An -N16 -tx /dev/urandom | tr -d ' '
 # Taking advantage of this feature allows us to launch our client directly
 # from the SIGUSR1 handler and avoid the need to poll for server readiness.
 trap 'DISPLAY=:$tty "${@:-$cfgdir/sxrc}"' USR1
-(trap '' USR1 && exec /usr/lib/xorg-server/Xorg :"$tty" -keeptty vt"$tty" -noreset -auth "$XAUTHORITY") & pid=$!
+(trap '' USR1 && exec /usr/lib/Xorg :"$tty" -keeptty vt"$tty" -noreset -auth "$XAUTHORITY") & pid=$!
```
Comment (Earnestly) [inline sx]: Change `exec /usr/lib/Xorg` to `exec Xorg`.

**Comment by Earnestly** [comment]
Tidy, thank you very much

**Comment by nameless3** [comment]
glad to help!

**Pull request merged**